### PR TITLE
[ENH] update validator page

### PR DESCRIPTION
**Pull request by Remi-Gau**
- closes https://github.com/bids-standard/bids-website/issues/491

### unrelated
- changes the 'spec' to 'specification'

### CODE REVIEW

File: docs/tools/validator.md
Comment (Remi-Gau) [inline docs/tools/validator.md:1]: Main change of the PR

Comment (Remi-Gau) [reply docs/tools/validator.md:1]: HTML version

https://bids-website--572.org.readthedocs.build/en/572/tools/validator.html

Comment (Remi-Gau) [reply docs/tools/validator.md:1]: @effigies maybe have a quick look at the html page see if there is anything I should add for now.

File: docs/tools/validator.md [hunk 1]
```diff
@@ -36,126 +36,91 @@ As such, there is no file uploading as part of the validation.
...
+            └── sub-01_ses-01_trc-CIMBI36_recording-manual_blood.tsv
 ```
 
-Now simply point the bids validator at the folder path of the subject(s) in
-question:
+Now simply point the bids validator at the folder path of the subject(s) in question:
 
 ```bash
-user@host:~/bids-examples$ bids-validator sub001
+user@host:~/bids-examples$ bids-validator pet001
```
Comment (effigies) [inline docs/tools/validator.md:88]: I would probably prefer to give an accurate command, though I can see the argument that simple might be better here:

```suggestion
user@host:~/bids-examples$ deno run -ERN jsr:@bids/validator pet001
```

**Pull request merged**